docs/csharp/language-reference/compiler-options/how-to-set-environment-variables-for-the-visual-studio-command-line.md

```diff
@@ -40,20 +40,24 @@ translation.priority.ht:
   - "zh-tw"
 ---
 # How to: Set Environment Variables for the Visual Studio Command Line
-The vsvars32.bat file sets the appropriate environment variables to enable command-line builds. For more information about vsvars32.bat, see [Knowledge Base article Q248802](http://go.microsoft.com/fwlink/?LinkId=225042).  
+
+The VsDevCmd.bat file sets the appropriate environment variables to enable command-line builds. For more information about VsDevCmd.bat, see [Knowledge Base article Q248802](http://go.microsoft.com/fwlink/?LinkId=225042).  
+
+> [!NOTE]
+> The VsDevCmd.bat file is a new file delivered with Visual Studio 2017. Visual Studio 2015 and earlier versions used VSVARS32.bat for the same purpose. This file was stored in Program Files\Microsoft Visual Studio\*Version*\Common7\Tools or Program Files (x86)\Microsoft Visual Studio\*Version*\Common7\Tools.
```

Comment:
this is rendering slightly funny. probably not your intention:
![image](https://user-images.githubusercontent.com/12971179/31039683-83875d20-a534-11e7-99c3-c5972e220839.png)

Reply:
Each occurrence of "Program Files" should have a backslash in front of it (it's a subdirectory off the root).